renderer/components/Onboarding/Steps/ConnectionType.js

```diff
@@ -2,22 +2,15 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { FormattedMessage } from 'react-intl'
 import { withFieldApi } from 'informed'
-import styled from 'styled-components'
 import { Box, Flex } from 'rebass'
 import { Form, RadioGroup } from 'components/UI'
 import ZapLogo from 'components/Icon/ZapLogoBolt'
-import BaseConnectionTypeItem from './ConnectionTypeItem'
+import BaseConnectionTypeItem from './components/ConnectionTypeItem'
+import Container from './components/Container'
 import messages from './messages'
 
 const ConnectionTypeItem = withFieldApi('connectionType')(BaseConnectionTypeItem)
 
-const Container = styled(Flex)`
-  position: absolute;
-  top: -30px;
-  bottom: 0;
-  visibility: ${props => (props.lndConnect ? 'hidden' : 'visible')};
```

Comment:
Removing the `visibility` based on `props.lndConnect` causes the steps to flash briefly when using an lnd connect link.

Reply:
resolved